buildlib/jucx/jucx-test.yml

```diff
@@ -32,8 +32,18 @@ jobs:
           ./contrib/configure-devel --prefix=$(Build.Repository.LocalPath)/install \
             --with-java --enable-gtest=no --with-cuda=$have_cuda
           make -j`nproc`
+          # Rename libjucx.so per platform and relink
+          arch=$(get_arch)
+          mv $(Build.Repository.LocalPath)/bindings/java/resources/libjucx.so \
+             $(Build.Repository.LocalPath)/bindings/java/resources/libjucx_"${arch}".so
+          mv $(Build.Repository.LocalPath)/bindings/java/src/main/native/build-java/test-classes/libjucx.so \
+             $(Build.Repository.LocalPath)/bindings/java/src/main/native/build-java/test-classes/libjucx_"${arch}".so
+          pushd $(Build.Repository.LocalPath)/bindings/java/src/main/native/.libs/
+          ln -s libjucx.so.0.0.0  libjucx_"${arch}".so
```

Comment:
why need to create SO with ${arch} for testing step?

Reply:
Removing...

Reply:
Added it back. Renaming the SO is required, so it would match the change in NativeLibs.java - it tries to load libjucx_${arch}.
Tests fail otherwise:
`java.lang.NoClassDefFoundError: Could not initialize class org.openucx.jucx.ucp.UcpConstants`